tools/sgbackstagecoverage/tools.go

```diff
@@ -0,0 +1,44 @@
+package sgbackstagecoverage
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/tools/sggcov2lcov"
+)
+
+// PostGoCoverage posts Go code coverage to the Backstage API.
+// apiURL is the URL to the API, for example 'https://api.mybackstagedeployment.com'.
+// entity is the entity ref, for example 'component:default/my-service'.
+// authEnv is the env var with the Bearer, for example 'MY_BACKSTAGE_TOKEN'.
+func PostGoCoverage(ctx context.Context, apiURL, entity, authEnv string) error {
```

Comment:
What do you think about passing the token directly and letting the caller decide whether that lives in an environment variable or a file or something else?

Reply:
Moved it to an argument instead, and also removed the dependency on the `sggcov2lcov` command making this a pure LCOV post command.